src/components/Pill/Listbox.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+const Listbox = (props) => {
+  const { children, className, label, ...rest } = props;
+
+  const sldsClasses = [
+    'slds-listbox',
+    'slds-listbox_horizontal',
+    className,
+  ];
+
+  const renderChild = ((child, i) => {
+    const clonedChild = React.cloneElement(child, {
+      'aria-selected': true,
+      tabIndex: i === 0 ? 0 : null,
+    });
+
+    return <li key={i} role="presentation">{clonedChild}</li>;
```

Comment:
Every time you put JS and JSX on the same line, a cute kitten drowns in its own tears.

Just saying.